src/Parallel/ParallelComponentHelpers.hpp

```diff
@@ -141,6 +142,17 @@ using get_option_tags = tmpl::remove_duplicates<tmpl::flatten<tmpl::transform<
     InitializationTagsList,
     detail::get_option_tags_from_initialization_tag<tmpl::_1>>>>;
 
+/// \ingroup ParallelGroup
+/// \brief Given a list of simple tags and a tagged tuple containing items
+/// created from input options, return a tagged tuple of items constructed
+/// by calls to create_from_options
+template <typename... SimpleTags, typename... OptionTags>
+tuples::TaggedTuple<SimpleTags...> get_from_options(
```

Comment:
I suggest to name this `create_from_options` to make clear that nontrivial work is being done when calling this function.

Also, can you make the `SimpleTags` an explicit template parameter instead of the `meta` argument? That would improve readability I think.

Reply:
okay to the rename; as the function is only called from `Main` I don't think it is worth the wrapping required to eliminate the `meta` argument

Reply:
I agree.